File: coding2025/Programers/level1/Java/이상한문자만들기.java
package coding2025.Programers.level1.Java;

public class 이상한문자만들기 {

    public String solution(String s) {

        String ans = "";
        int cnt = 0;
        String[] array = s.split("");

        for(String ss : array){
            cnt = ss.contains(" ") ? 0 : cnt+1;
            ans += cnt%2==0 ? ss.toLowerCase() : ss.toUpperCase();
        }

        return ans;
    }
}

File: AlgoPython/lev1/이상한문자만들기.py
def solution(s):
    ans = ''
    new_list = s.split(' ')

    for i in new_list:
        for j in range(len(i)):
            if j%2 == 0:
                ans += i[j].upper()
            else:
                ans += i[j].lower()

        ans += ' '

    return ans[0:-1]